backend/rust-gbt/src/gbt.rs

```diff
@@ -35,10 +36,12 @@ impl PartialEq for TxPriority {
 impl Eq for TxPriority {}
 impl PartialOrd for TxPriority {
     fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
-        if self.score == other.score {
-            Some(self.uid.cmp(&other.uid))
-        } else {
+        if self.score != other.score {
             self.score.partial_cmp(&other.score)
+        } else if self.order != other.order {
+            Some(other.order.cmp(&self.order))
```

Comment:
Double checking: order cmp value is flipped compared to others.

Reply:
yep, I should have added a comment - transactions are ordered by *descending* score but *ascending* txid.